esphomeyaml/components/sensor/total_daily_energy.rst

```diff
@@ -31,6 +31,7 @@ daily energy usage in ``Wh`` or ``kWh``.
     time:
       - platform: sntp
         id: my_time
+        timezone: Europe/Amsterdam
```

Comment:
This should not be required. The auto-detection should handle this - if not, that's a bug and not an issue for the docs.